components/PlaylistDetails.tsx

```diff
@@ -0,0 +1,44 @@
+import { SpotifyPlaylist } from '@/models/Spotify';
+import Image from 'next/image';
+
+interface Props {
+  playlist?: SpotifyPlaylist;
+  isLoading?: boolean;
+}
+
+const PlayListDetails: React.FC<Props> = ({ playlist, isLoading }) => {
+  if (isLoading) {
+    return (
+      <div className="animate-pulse flex flex-col items-center">
+        <div className="w-full rounded-full bg-gray-600 circle" />
+        <div className="h-4 w-8/12 bg-gray-600 mt-2" />
+        <style jsx>{`
+          .circle::before {
+            content: '';
+            display: block;
+            padding-bottom: 100%;
+          }
+        `}</style>
+      </div>
+    );
+  }
+
+  return (
+    <div className="text-center">
+      {playlist.images.length > 0 && (
+        <Image
+          src={playlist.images[0]?.url}
+          width={208}
+          height={208}
+          className="rounded-full"
```

Comment:
Can we keep these playlist image square? I think it's more beneficial since most of the images show a snippet of four different albums. It will allow the user to see those fully without using the rounded borders. Once we make this change, make sure to update the skeleton as well to be square.